src/main/java/org/openlmis/buq/service/buq/BottomUpQuantificationService.java

```diff
@@ -295,6 +305,14 @@ BottomUpQuantification findBottomUpQuantification(UUID bottomUpQuantificationId)
         );
   }
 
+  Remark findRemark(UUID remarkId) {
+    return remarkRepository
+        .findById(remarkId)
+        .orElseThrow(() -> new ContentNotFoundMessageException(
+            ERROR_REMARK_NOT_FOUND, remarkId)
+        );
+  }
+
```

Comment:
with what I said above, I meant that in the `RemarkService` you already have `findOne` method.

Reply:
I changed repository usage to service.